src/state/slices/opportunitiesSlice/types.ts

```diff
@@ -12,12 +12,64 @@ export type AssetIdsTuple =
   | readonly [AssetId]
   | readonly []
 
-export type OpportunityMetadata = {
+/**
```

Comment:
Copied from the `types` implementation, do we want to export these instead?

Reply:
Yep, I think so. Either that, or move them higher up in the `web` tree - they don't belong in the opportunities slice.

Reply:
3f3b801c5e0ae53d18fb8a532c443d8a4c905fd9